benchmarks/solve.py

```diff
@@ -54,3 +54,31 @@ def time_solve(self):
 
     def time_solve_nocheck(self):
         sympy.solve(self.eqs, *self.p.c, check=False)
+
+
+class TimeMatrixSolve:
+
+    params = ['GE', 'LU' 'ADJ']
+
+    def setup(self, name):
+
+        n = 3
+
+        self.A = sympy.Matrix(n, n,
+                              lambda i, j: sympy.Symbol('a{}{}'.format(i, j)))
+        self.b = sympy.Matrix(n, 1,
+                              lambda i, j: sympy.Symbol('b{}{}'.format(i, j)))
+        self.A_sym = sympy.Matrix(n, n, lambda i, j:
+                                  sympy.Symbol('a{}{}'.format(*sorted((i, j)))))
+
+    def time_solve(self, name):
+
+        self.A.solve(self.b, method=name)
+
+    def time_lusolve(self, name):
```

Comment:
time_lusolve and time_cholesky_solve should be moved to a separate class (since they aren't affected by the parameter)